notes/Module-2/2.2.5-Unit_2_Knowledge_quiz.md

```diff
@@ -0,0 +1,12 @@
+# Coach Notes
+
+## Coach comments
+Please add comments below which will help future coaches delivering this topic or suggestions for improvement of the content.
+
+|**Coach name**|**Delivery date**|**Comments**|
+|--------------|-----------------|------------|
+|Mandy|13/05/21|The following questions need reviewing: 8 - I thought the answer was URL Encoding, not sure URI Encoding exists?, 14 - sneaker database is not accessible now with a key, 16 & 17 - my understanding of static testing is pre-compilation time i.e. when you use tools like Sonar or manual code reviews to run static analysis of code. What is described is to me, unit tests and integration tests.|
```

Comment:
Yes the sneaker database is now behind RapidAPI paywall :( would need to find another open API?

Reply:
<img width="614" alt="Screenshot 2021-05-25 at 11 09 52" src="https://user-images.githubusercontent.com/4499581/119480610-c0cf3780-bd49-11eb-8f08-e7ea37f97fa1.png">

Reply:
URI encoding is a thing

Reply:
yes there is a JS function called encodeURI but in Java its URLEncoder and other sites all seem to refer to URLEncoding e.g. https://www.w3schools.com/tags/ref_urlencode.ASP. I think if we just remove  one of URL encoded or URI encoded we'll save any confusion :-)